### ISSUE TITLE
Handoff in materializer_vnode

### ISSUE THREAD

**Issue by deepthidevaki** [SyncFree/antidote#203]
After handoff, the operations may be applied twice. This is because, logging_vnode does the handoff and gets the operations to the log. materializer_vnode does the handoff independently, and caches the operations. Then (for some executions) the load_from_log in the materilizer_vnode re-loads the operation that is already cached by the handoff, thus duplicating them.
The test is in the branch handoff_materiliazer riak_test/log_handoff_test.erl.

**Comment by tcrain** [comment]
Ok, it looks like I introduced this when add the recovery from the log, so I will fix it.  The test you mention will fail on master?  Can I use it to test if the fix I will make works?

**Comment by deepthidevaki** [comment]
Yes. It fails on master. It is the same test that is in the master, but with op-crdts. The bug is hidden with state based crdts.
